models/Document/Dao.php

```diff
@@ -206,7 +206,7 @@ public function updateWorkspaces()
      *
      * @return array
      */
-    public function updateChildsPaths($oldPath)
+    public function updateChildrenPaths($oldPath)
```

Comment:
Why not `updateChildPaths` seems like that would be more "correct" English (see also line 219 below)?